src/runtime/storageNG/unified-store.ts

```diff
@@ -58,7 +57,7 @@ export function entityHasName(name: string) {
  * Once the old-style stores are deleted, this class can be merged into the new
  * Store class.
  */
-export abstract class UnifiedStore implements Comparable<UnifiedStore>, OldStore {
+export abstract class UnifiedStore implements Comparable<UnifiedStore> {
   // Tags for all subclasses of UnifiedStore.
   protected abstract unifiedStoreType: 'Store' | 'StorageProviderBase';
```

Comment:
optional: remove 'StorageProviderBase' (I imagine you are going to clean up this class anyway)